scripts/create-clusters.sh

```diff
@@ -25,7 +25,7 @@ set -o pipefail
 source "${BASH_SOURCE%/*}/util.sh"
 NUM_CLUSTERS="${NUM_CLUSTERS:-2}"
 KIND_IMAGE="${KIND_IMAGE:-}"
-KIND_TAG="${KIND_TAG:-}"
+KIND_TAG="${KIND_TAG:-v1.19.1@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600}"
```

Comment:
Let's take advantage and bump to latest:
```suggestion
KIND_TAG="${KIND_TAG:-v1.19.4@sha256:796d09e217d93bed01ecf8502633e48fd806fe42f9d02fdd468b81cd4e3bd40b}"
```

Will need to update https://github.com/kubernetes-sigs/kubefed/blob/ecd49000b586e2ed29e3a1b938e2f85772358843/scripts/pre-commit.sh#L139 too.